### ISSUE TITLE
Kind=yes should be dropped

### ISSUE THREAD

**Issue by nvkelso** [mapzen/vector-datasource#335]
Did we regress the code the dropped `*=yes` from OSM (so `kind=yes` doesn't get generated in tiles)?

http://vector.dev.mapzen.com/osm/all/18/41885/101371.topojson?api_key=vector-tiles-HqUVidw

```
{
type: "Point",
properties: {
kind: "yes",
id: 3575638173,
name: "Special Education & Communicative Disorders"
},
coordinates: [
252,
94
]
}
]
```

**Comment by nvkelso** [comment]
Related: https://github.com/mapzen/vector-datasource/issues/282

**Comment by zerebubuth** [comment]
What's happening is that our queries will accept either `lock=yes` or `office=yes` as acceptable values, and the context is stripped in the call to `COALESCE`. As you say, related to #282.

We have a few options, including:

1. Simplest is to just ignore `lock=yes` and `office=yes`. Note that this will drop locks entirely, as `yes` is the only value we query for them. But we could come back to it later.
2. Replace `*=yes` with `kind=*`. This would mean complexifying the query a little.
3. Pull all the values `"aerialway", ... "waterway"` back to Python-land and normalise them to a kind there. This is, I think, what we'll end up doing for #282 anyway to have more control over kind selection.

@nvkelso What do you reckon? Quick fix for now, and revisit as part of #282, or might as well start cleaning it up now?

**Comment by nvkelso** [comment]
In this example, it looks like this is not a regression from `prod` v0.4.x tiles (e.g.: the same problem is seen in http://vector.mapzen.com/osm/all/18/41885/101371.topojson?api_key=vector-tiles-HqUVidw). Not sure what the state was in v0.3.0 and earlier tiles.

Since it doesn't appear to be a regression, I vote for general fix as part of #282, and I agree option 3 you outline above is probably where we'd end up anyhow. (Option 1 isn't an option since we'd regress features.)

Closing this issue and copying over some info here to that original issue.
